docs/_getting-started-linux-android.md

```diff
@@ -120,77 +91,4 @@ If you have recently installed Android Studio, you will likely need to [create a
 
 > We recommend configuring [VM acceleration](https://developer.android.com/studio/run/emulator-acceleration.html#vm-linux) on your system to improve performance. Once you've followed those instructions, go back to the AVD Manager.
 
-Click "Next" then "Finish" to create your AVD. At this point you should be able to click on the green triangle button next to your AVD to launch it, then proceed to the next step.
-
-<h2>Running your React Native application</h2>
-
-<h3>Step 1: Start Metro</h3>
-
-[**Metro**](https://metrobundler.dev/) is the JavaScript build tool for React Native. To start the Metro development server, run the following from your project folder:
-
-<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
-<TabItem value="npm">
-
-```shell
-npm start
-```
-
-</TabItem>
-<TabItem value="yarn">
-
-```shell
-yarn start
-```
-
-</TabItem>
-</Tabs>
-
-:::note
-If you're familiar with web development, Metro is similar to bundlers such as Vite and webpack, but is designed end-to-end for React Native. For instance, Metro uses [Babel](https://babel.dev/) to transform syntax such as JSX into executable JavaScript.
-:::
-
-<h3>Step 2: Start your application</h3>
-
-Let Metro Bundler run in its own terminal. Open a new terminal inside your React Native project folder. Run the following:
-
-<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
-<TabItem value="npm">
-
-```shell
-npm run android
-```
-
-</TabItem>
-<TabItem value="yarn">
-
-```shell
-yarn android
-```
-
-</TabItem>
-</Tabs>
-
-If everything is set up correctly, you should see your new app running in your Android emulator shortly.
-
-This is one way to run your app - you can also run it directly from within Android Studio.
-
-> If you can't get this to work, see the [Troubleshooting](troubleshooting.md) page.
-
-<h3>Modifying your app</h3>
-
-Now that you have successfully run the app, let's modify it.
-
-- Open `App.tsx` in your text editor of choice and edit some lines.
-- Press the <kbd>R</kbd> key twice or select `Reload` from the Dev Menu (<kbd>Ctrl</kbd> + <kbd>M</kbd>) to see your changes!
-
-<h3>That's it!</h3>
-
-Congratulations! You've successfully run and modified your first React Native app.
-
-<center><img src="/docs/assets/GettingStartedCongratulations.png" width="150"></img></center>
-
-<h2>Now what?</h2>
-
-- If you want to add this new React Native code to an existing application, check out the [Integration guide](integration-with-existing-apps.md).
-
-If you're curious to learn more about React Native, check out the [Introduction to React Native](getting-started).
```

Comment:
I feel like this part was important, or at least this updated doc page looks truncated if we don't include an ending